README.md

```diff
@@ -40,3 +40,74 @@ if ($file->getSize()) {
 // Access a resource provided by this Divide.IQ server.
 $result = $divideIq->request('stockbase_stock');
 ```
+
+Example Usage orderRequest POST
+=============
```

Comment:
Fixed in commit https://github.com/Nav-Appaiya/PHPDivideIQ/commit/9c4fa5e6ca375d48add369eeb2b75f072dc14e3f and https://github.com/Nav-Appaiya/PHPDivideIQ/commit/90b6d707b39272a6d9253ca37212ac69d4ec3da9